src/Datadog.Trace/Datadog.Trace.nuspec

```
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<package  xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>    
+    <id>Datadog.Trace</id>
+    <version>1.7.0</version>
+    <title>Datadog APM</title>
+    <authors>lucas.pimentel.datadog;colinhigginsdatadog;zachmontoyadd</authors>
+    <owners>Datadog</owners>
+    <licenseUrl>https://github.com/DataDog/dd-trace-dotnet/blob/master/LICENSE</licenseUrl>
+    <projectUrl>https://docs.datadoghq.com/tracing/setup/dotnet/</projectUrl>
+    <iconUrl>https://imgix.datadoghq.com/img/favicons/favicon-32x32.png</iconUrl>
```

`iconUrl` is also deprecated, but you can just use this 64x64 icon for now. We should fix it here and in [`src/Directory.Build.props`](https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Directory.Build.props) at some point.
```suggestion
    <iconUrl>https://github.com/DataDog/dd-trace-dotnet/raw/master/datadog-logo-64x64.png</iconUrl>
```

---

+1 for fixing the deprecated `iconUrl` at a later point